docs/guide/performance.md

```diff
@@ -12,9 +12,11 @@ Vite's internal and official plugins are optimized to do the least amount of wor
 
 However, the performance of community plugins is out of Vite's control, which may affect the developer experience. Here are a few things you can look out for when using additional Vite plugins:
 
-1. The `buildStart`, `config`, and `configResolved` hooks should not run long and extensive operations. These hooks are awaited during dev server startup, which delays when you can access the site in the browser.
+1. Large dependency that is only used in rare cases or only used in either dev or build should not be statically imported. These are some examples that reduced the config load time by changing static imports to dynamic ones: [vite-plugin-react#212](https://github.com/vitejs/vite-plugin-react/pull/212), [vite-plugin-pwa#224](https://github.com/vite-pwa/vite-plugin-pwa/pull/244).
```

Comment:
```suggestion
1. Large dependencies that are only used in certain cases should be dynamically imported to reduce the Node.js startup time. Example refactors: [vite-plugin-react#212](https://github.com/vitejs/vite-plugin-react/pull/212) and [vite-plugin-pwa#224](https://github.com/vite-pwa/vite-plugin-pwa/pull/244).
```

Perhaps this is a bit more concise. I also referred it to "Node.js startup time" instead of "config load time" since it's more direct as to where the bottleneck is?